ui/root/src/commonMain/kotlin/social/androiddev/root/splash/SplashContent.kt

```diff
@@ -56,12 +59,14 @@ fun SplashContent(
     ) {
         Text("Loading")
 
-        LaunchedEffect(Unit) {
-            // TODO: Hook up to a DI ViewModel
-            if (true) {
+        LaunchedEffect(authStatus) {
+            if (authStatus is AuthStatus.Authorized) {
+                // TODO(krzysztof): Do not navigate to timeline, until logout func is implemented
```

Comment:
This is because there's no way to navigate back to Welcome screen, if there's a problem fetching the timeline.